GVFS/GVFS.Common/Git/GitObjects.cs

```diff
@@ -42,6 +42,11 @@ public enum DownloadAndSaveObjectResult
             Error
         }
 
+        public static bool IsLooseObjectsDirectory(string value)
+        {
+            return value.Length == 2 && !value.Any(c => !Uri.IsHexDigit(c));

```

Comment:
Also update `SHA1Util.IsValidShaFormat` to use `IsHexDigit`?

Reply:
I could, but does it seem weird for it to be in this PR now that I'm not using any SHAUtil functions.  I'm happy to do it if you are to approve :-)

Reply:
I am OK with whichever way you decide to go :) My thinking was that we might as well update the code to be consistent, but I am okay leaving it as-is as well.

Reply:
I would change this to use `All` so that it doesn't have double negatives.
```
return value.Length == 2 && value.All(c => Uri.IsHexDigit(c));
```